Student/Student/Courses/CourseList/CourseListViewController.swift

```diff
@@ -52,6 +52,10 @@ class CourseListViewController: UIViewController, CourseListViewProtocol {
         // Collection View Setup
         collectionView?.refreshControl = refreshControl
 
+        let id = String(describing: CourseCardCell.self)
+        let nib = UINib(nibName: id, bundle: Bundle(for: type(of: self)))
+        collectionView?.register(nib, forCellWithReuseIdentifier: id)
```

Comment:
do we not have an extension for registering that mirrors the dequeue? That'd be nice.

Reply:
I'll take a look.